allennlp/tests/nn/util_test.py

```diff
@@ -781,6 +781,24 @@ def test_combine_tensors_and_multiply_with_same_batch_size_and_embedding_dim(sel
                             [[5 * 2 * 4 + 5 * 3 * 5, 2 * 2 * 4 + 2 * 3 * 5],
                              [4 * 1 * 4 + 4 * 1 * 5, 3 * 1 * 4 + 3 * 1 * 5]])
 
+    def test_combine_tensors_and_multiply_with_batch_size_one(self):
```

Comment:
A test of this case with batch size larger than one would have caught the error above, I think.